src/Components/Login.js

```diff
@@ -1,30 +1,139 @@
 import Button from 'react-bootstrap/Button';
 import Form from 'react-bootstrap/Form';
 import 'bootstrap/dist/css/bootstrap.min.css'
+import { useEffect, useState } from 'react';
+import AdminPanel from './adminPanel';
+import { json, useNavigate  } from 'react-router-dom';
+import { Route, Routes } from 'react-router-dom';
+import Dropdown from 'react-bootstrap/Dropdown';
+
+function Login() {
+
+    const navigate= useNavigate();
+
+     const [Email, setEmail] = useState('');
+     const [Password, setPassword] = useState('');
+     const [Loggedin, SetLoggedin]= useState(false);
+     let selectedvalue;
+
+     const [selectedOption, setSelectedOption] = useState('0');
+
+    const handleOptionChange = (event) => {
+        setSelectedOption(event.target.value);
+        selectedvalue=event.target.value;
+        console.log(selectedvalue);
+    };
+
+
+    //  useEffect(()=>{
+
+    //         return(
+    //             <div>
+    //                 {<AdminPanel/>}
+    //             </div>
+    //         )
+
+    //  },[Loggedin])
+
+    const handleSubmit = (event) => {
+        event.preventDefault();
+        if(selectedOption=="0")
+        {
+            fetch('http://localhost:3000/api/v1/admins/login', {
```

Comment:
I think there is a better approach than writing the base url repeatedly. You can write the base url in .env file, or write a component which will return the base url and name export the component.